cmake_modules/FindZMQ.cmake

```diff
@@ -0,0 +1,49 @@
+# - Try to find ZMQ
+# Once done this will define
+#
+#  ZMQ_FOUND - system has ZMQ
+#  ZMQ_INCLUDE_DIR - the ZMQ include directory
+#  ZMQ_LIBRARIES - Link these to use ZMQ
+
+if (ZMQ_INCLUDE_DIR AND ZMQ_LIBRARIES)
+
+  # in cache already
+  set(ZMQ_FOUND TRUE)
+  message(STATUS "Found libzmq: ${ZMQ_LIBRARIES}")
+
+else (ZMQ_INCLUDE_DIR AND ZMQ_LIBRARIES)
+
+  find_path(ZMQ_INCLUDE_DIR
+    NAMES zmq.h
+    ${_obIncDir}
+    ${GNUWIN32_DIR}/include
+  )
+
+  find_library(ZMQ_LIBRARIES
+    NAMES libzmq3
```

Comment:
There is no such library. Please change this to **zmq**

Reply:
Fixed thanks